src/utils/viewer/viewer.cpp

```diff
@@ -903,6 +903,9 @@ void Viewer::initSimPostReconfigure() {
     }
   } else if (recomputeNavmesh_) {
     esp::nav::NavMeshSettings navMeshSettings;
+    navMeshSettings.agentHeight = agentConfig_.height;
+    navMeshSettings.agentRadius = agentConfig_.radius;
+    navMeshSettings.agentMaxSlope = 55.0f;
```

Comment:
Danger here is that users will load the asset in viewer and everything looks good, but using the defaults in the episode generator for training won't produce the same results.